.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
In my experience, using a daily interval like this quickly becomes annoying, unless you really don't have that many dependencies. It seems like that might be the case, though, since there's only 1 pretty small file (`.github/workflows/test.yml`).